connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java

```diff
@@ -0,0 +1,67 @@
+package io.camunda.connector.runtime.inbound.webhook;
+import io.camunda.connector.api.inbound.InboundConnectorContext;
+import io.camunda.connector.impl.inbound.InboundConnectorProperties;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConditionalOnProperty("camunda.connector.webhook.enabled")
+public class WebhookConnectorRegistry {
+
+  public static final String TYPE_WEBHOOK = "io.camunda:webhook:1";
```

Comment:
I am a bit confused here. You specify type webhook as constant. Will the arbitrary webhooks work?
For example, existing [HTTP](https://github.com/camunda/connectors-bundle/blob/main/connectors/webhook-connector/element-templates/webhook-connector.json#L4) and [GitHub](https://github.com/camunda/connectors-bundle/blob/main/connectors/github/element-templates/github-webhook-connector.json#L4)?

Reply:
Yes, they will: [previously](https://github.com/camunda-community-hub/spring-zeebe/blob/5dd18aeb23c10f29167a12261f6800253c6f17f9/connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/registry/InboundConnectorProperties.java#L23) it was also defined as a constant, but it was simply `webhook.` I changed it to match the type pattern that we already have for all outbound connectors.

For custom templates to work, [this property](https://github.com/camunda/connectors-bundle/blob/0655593e5c84223bd2a4ddf257f8e5790a0eafd6/connectors/webhook-connector/element-templates/webhook-connector.json#L32) will need to be changed from `webhook` to `io.camunda:webhook:1`.